decisiontree/tests/test_app.py

```diff
@@ -0,0 +1,152 @@
+from model_mommy import mommy
+
+from django.conf import settings
+from django.core import mail
+
+from rapidsms.messages.incoming import IncomingMessage
+
+from decisiontree.multitenancy import models as link_models
+from decisiontree import tasks
+from decisiontree import models as dt
+
+from .cases import DecisionTreeTestCase
+
+
+class BasicSurveyTest(DecisionTreeTestCase):
+
+    def setUp(self):
+        super(BasicSurveyTest, self).setUp()
+        self.survey = mommy.make('decisiontree.Tree', trigger='food')
+        link_models.TreeLink.all_tenants.create(
```

Comment:
These could use the default manager. `link_models.TreeLink.objects.by_tenant(self.tenant).create(linked=self.survey)`. It doesn't matter too much, but it does enforce that a tenant should be provided, where the all_tenants manager doesn't.